### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by bariboo** [maximmax42/Discord-CustomRP#158] [Q&A] [answered]
Hi ! I would to desinsal CustomRP, but i don't know what I need to do, i'm in windows.

**Comment by maximmax42** [comment, marked as answer]
I assume you mean uninstall. First, close the app either by going File -> Quit or by right clicking tray icon and choosing Quit. Then you can search for "Uninstall CustomRP" in Windows search and just run that.
